RELEASE_NOTES.md

```diff
@@ -10,6 +10,8 @@
   - KotlinSqlObjectPlugin will install forgotten SqlObjectPlugin for you
   - ClasspathSqlLocator allows disabling comment stripping and deprecate static API
   - KotlinMapper respects `@PropagateNull`
+  - Freemarker allows customizing Configuration
+  - FreemarkerSqlLocator should now be created via FreemarkerConfig so it shares config
```

Comment:
Part of a different PR?

Reply:
I did a `git merge master` into this branch to resolve the conflicts, as I didn't want to rebase across Marnes' work